traits.rs

```diff
@@ -119,3 +122,16 @@ pub trait PSP22 {
         delta_value: u128,
     ) -> Result<(), PSP22Error>;
 }
+
+#[ink::trait_definition]
+pub trait PSP22Metadata {
```

Comment:
I don't understand why metadata is not part of the PSP22 standard, I think it should. 

If it isn't then I'd put it behind a feature flag as well. As with other feature flags - if they're not needed we shouldn't force people to download its code.

Reply:
Same here, I think it should be a part of standard, especially considering the fact that name and symbol return Option and decimals could always be defined (and default to 0).

That being said, I don't think this will cause any additional bytes in a compiled contract if not hidden behind a feature flag. Traits are a compile-time type safety feature, importing `PSP22` or `PSP22Metadata` trait in your contract imposes some "rules" on your code, but does not make it bigger (I think?). Someone using this crate will choose which parts they need simply with import statements: `use psp22::PSP22;` or `use psp22::{PSP22, PSP22Metadata};` Hiding one tiny trait behind a feature flag seems to me like an unnecessary annoyance.

Reply:
I think every token should implement Metadata. It's really important for indexers, wallets, and offchain tooling in general

Reply:
If it was just my decision, I would merge `PSP22Metadata` trait into `PSP22` and require each PSP22 token to feature these 3 info-methods. But that would mean going beyond the standard and being kinda highly opinionated, so not sure if this is a good idea. But I'm not completely against doing that...

Reply:
Yeah, this would probably no be ideal, again because of selectors. The old selectors that are used are `PSP22Metadata::symbol` and not `PSP22::symbol` :/ So probably it's better to keep it as it is...